src/main/java/at/hannibal2/skyhanni/features/event/hoppity/HoppityRabbitRarity.kt

```diff
@@ -0,0 +1,18 @@
+package at.hannibal2.skyhanni.features.event.hoppity
+
+enum class HoppityRabbitRarity(val displayName: String, val colorCode: String) {
```

Comment:
is there a reason for this map? LorenzRarity should get used here. the two special functions below can both move into their classes where they are used

Reply:
Yeah, just noticed https://github.com/hannibal002/SkyHanni/blob/f3b5eaf245aa4975ff4a3787c7ec45f6799b369f/src/main/java/at/hannibal2/skyhanni/features/event/hoppity/HoppityEventSummary.kt#L271

Reply:
I _might_ be able to work around using LorenzRarity instead, and using some selective querying to do what I am right now, but would take some reworking